docs/docs/databases/presto.mdx

```diff
@@ -35,3 +35,16 @@ datasource. If you’re using an older version of Presto, you can configure it i
     "version": "0.123"
 }
 ```
+
+SSL Secure extra add json config to extra connection information.
+
+``` 
+   {
+     "connect_args":
+    {"protocol": "https",
+     "requests_kwargs":{"verify":false}
+  } 
+}
+```
+
+
```

Comment:
You need to remove this trailing whitespace:
![image](https://github.com/apache/superset/assets/33317356/59dfac4f-61d2-4adc-870d-75dfe0067d69)